investment-analysis-on-edgar-using-vss/README.md

```diff
@@ -0,0 +1,80 @@
+# Investment Analysis using Vector Similarity in Amazon Neptune Analytics - Demo
+
+This repository contains a practical demo showcasing the concepts discussed in the "Investment Analysis using Vector Similarity in Amazon Neptune Analytics" blog series. The demo covers setting up a data pipeline to convert the raw EDGAR dataset to a Knowledge Graph, running Vector Similarity Search (VSS) combined with graph traversal queries, and provides a frontend for visualization.
+
+## Source of Original Data
+The original data can be accessed from the U.S. Securities and Exchange Commission's EDGAR database at SEC's EDGAR Search and Access.
+
+## Sourcing Descriptions for Holder Vertex
+The descriptions for the holder vertex in the knowledge graph are sourced using the BedrockAPI class. This involves generating detailed descriptions for each business entity within the holder vertex using advanced language models on Bedrock. Specifically, the Claude-v2 model is leveraged to provide rich, contextual descriptions that are then added to a new column labeled 'description:string'. These descriptions enrich the knowledge graph, aiding in deep analysis, trend identification, investment decision-making, and the discovery of unique business strategies.
+
+## Creation of Embeddings
+Embeddings within the holder vertex are created by the HolderNodeDescriptionEmbedding class. This class enhances the holder vertex by converting textual descriptions in the 'description:string' column into numerical vector form. It utilizes the "all-MiniLM-L6-v2" model from the Sentence Transformer suite to perform this encoding. The process transforms descriptive text into a format suitable for advanced computational analyses, such as similarity comparisons among top investors.
+
+## Raw EDGAR Data Format
```

Comment:
This is great.

Can you please provide the same header for the merged dataset that we curated?

Reply:
With the header of the curated dataset that we created for this; I approve this PR. Thank you, Angad!